<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayerClass</class>
 <widget class="QMainWindow" name="MediaPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mediaplayer.qrc">
    <normaloff>:/MediaPlayer/Resources/logo.ico</normaloff>:/MediaPlayer/Resources/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openLocal">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>540</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{border-image: url(:/MediaPlayer/Resources/open_normal.png);}
QPushButton:hover{border-image: url(:/MediaPlayer/Resources/open_hot.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseCon">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>540</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{border-image: url(:/MediaPlayer/Resources/play_normal.png);}
QPushButton:hover{border-image: url(:/MediaPlayer/Resources/play_hot.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rtsp播放地址：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getRtsp">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>20</y>
      <width>111</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/MediaPlayer/Resources/get.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totalTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="playTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>00:00/</string>
    </property>
   </widget>
   <widget class="QSlider" name="playSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>711</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mediaplayer.qrc"/>
 </resources>
 <connections/>
</ui>
